packages/vuetify/src/components/VBottomNavigation/VBottomNavigation.tsx

```diff
@@ -0,0 +1,107 @@
+// Styles
+import './VBottomNavigation.sass'
+
+// Composables
+import { makeBorderProps, useBorder } from '@/composables/border'
+import { makeDensityProps, useDensity } from '@/composables/density'
+import { makeElevationProps, useElevation } from '@/composables/elevation'
+import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
+import { makePositionProps, usePosition } from '@/composables/position'
+import { makeRoundedProps, useRounded } from '@/composables/rounded'
+import { makeTagProps } from '@/composables/tag'
+import { useBackgroundColor } from '@/composables/color'
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Utilities
+import { computed, defineComponent, toRef } from 'vue'
+import { convertToUnit } from '@/util'
+import { makeProps } from '@/util/makeProps'
+
+export default defineComponent({
+  name: 'VBottomNavigation',
+
+  props: makeProps({
+    color: String,
+    flat: Boolean,
+    grow: Boolean,
+    height: {
+      type: [Number, String],
+      default: 56,
+    },
+    horizontal: Boolean,
```

Comment:
I thought the point of the v-btn changes with `stacked` was that it would be used for situations like this, so horizontal would not be needed?

Reply:
We can have this discussion once we get to tabs as it makes sense to have icons stack by default in all implementations.